process-deploy/src/main/java/com/wipro/ats/bdre/pdeploy/ProcessDeployMain.java

```diff
@@ -24,9 +24,12 @@
  * main class to run ProcessDeploy
  */
 public class ProcessDeployMain {
+
+    private ProcessDeployMain(){}
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)